packages/auth/src/AuthCard/AuthCard.tsx

```diff
@@ -33,66 +33,35 @@ export const AuthCard = ({
   const { logo } = React.useContext(LogoContext);
 
   return (
-    <Card
-      variant="primary"
-      sx={{
-        maxWidth: '340px',
-        fontFamily: 'body',
-        padding: 7,
-        paddingTop: 8,
-        border: 'none',
-        background: 'white',
-      }}
-    >
+    <Card variant="forms.auth.cardContainer">
       <Flex sx={{ flexDirection: 'column', gap: 3 }}>
         {logo && (
           <Flex sx={{ width: '100%', justifyContent: 'center' }}>{logo}</Flex>
         )}
 
         {children}
 
-        <Flex sx={{ justifyContent: 'space-between', marginTop: 7 }}>
+        <Flex variant="forms.auth.submitButtonContainer">
```

Comment:
In this specific case that is only from Auth module, we can use `sx` because we know that it won't need to change.